compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -257,7 +257,10 @@ def start_endpoint(
         # potentially running with the existing pidfile
         if pid_check["exists"]:
             if pid_check["active"]:
-                log.info("Endpoint is already active")
+                endpoint_name = endpoint_dir.name
+                if endpoint_config.display_name:
+                    endpoint_name = endpoint_config.display_name
+                print(f"Endpoint '{endpoint_name}' is already active")
```

Comment:
I'm not sure how the MT story handles things differently than the ST, but I initially removed the log.info as this is most likely just a no-op log noise ssuming things are working normally.  I suppose there are error conditions we don't foresee (like MT) where this would help debug an issue.

In all other cases I added a print but kept the log messages.  Added the log.info back as it's no big deal.